tutor/commands/k8s.py

```diff
@@ -26,7 +26,13 @@ def __init__(self) -> None:
         # pylint: disable=import-outside-toplevel
         from kubernetes import client, config
 
-        config.load_kube_config()
+        try:
+            config.load_incluster_config()
```

Comment:
The problem I have with this implementation is that the default path will fail for most people -- who are not running Tutor inside a pod. 

Is there any other clue that could allow Tutor to be smart and guess that it's running inside a pod? For instance: an environment variable, a certain file, etc.?

Reply:
According to the kubernetes package source code, we should check for the following environment variables: `kubernetes.config.incluster_config.SERVICE_HOST_ENV_NAME`, `kubernetes.config.incluster_config.SERVICE_PORT_ENV_NAME`.
We should also check for the existence of the presence of the following file: `kubernetes.config.kube_config.KUBE_CONFIG_DEFAULT_LOCATION`.

Reply:
Exactly.

Reply:
I made a slight modification.
Initially, we will verify if the KUBE_CONFIG_DEFAULT_LOCATION exists. Next, we will check if the tutor is operating within a pod and load the incluster_config. If not, an error will be raised.